packages/test/test-service-load/src/utils.ts

```diff
@@ -31,7 +31,7 @@ class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
     private logs: ITelemetryBaseEvent[] = [];
 
     public constructor(private readonly baseLogger?: ITelemetryBufferedLogger) {
-        super();
+        super(undefined, {all:{testVersion: pkgVersion}});
```

Comment:
#nit Should we have a comment with the parameter name for the `undefined` parameter?